packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/animations/curveEditor/graph/rangeFrameBarComponent.tsx

```diff
@@ -128,7 +128,7 @@ export class RangeFrameBarComponent extends React.Component<IRangeFrameBarCompon
     }
 
     private _buildActiveFrame() {
-        if (this.props.context.activeFrame !== null && this.props.context.activeFrame !== undefined) {

```

Comment:
is that a long standing bug? it's completely flipped

Reply:
It was a bug from my previous ACE PR that I noticed and fixed now